performance_regression_test.py

```diff
@@ -797,3 +798,100 @@ def build_histogram(self, workload: PerformanceTestWorkload, test_type: Performa
 
         self.update_hdrhistograms(histogram_name='test_histogram_by_interval',
                                   histogram_data=histogram_data_by_interval)
+
+
+class PerformanceRegressionUpgradeTest(PerformanceRegressionTest, UpgradeTest):  # pylint: disable=too-many-ancestors
+    def __int__(self, *args):
+        super().__init__(*args)
+        self.cluster = None  # pylint: disable=attribute-defined-outside-init
+        self.tester = None  # pylint: disable=attribute-defined-outside-init
+        self.monitoring_set = None  # pylint: disable=attribute-defined-outside-init
+
+    def get_email_data(self):  # pylint: disable=no-self-use
+        return PerformanceRegressionTest.get_email_data(self)
+
+    @latency_calculator_decorator(legend="Upgrade Node")
+    def upgrade_node(self, node):  # pylint: disable=arguments-differ
+        InfoEvent(message='Upgrade Node %s begin' % node.name).publish()
```

Comment:
For this kind of functionality we have `log_run_info` decorator.
So, I think it must be reused here.

Reply:
removing this code block, as if this typo was already in there beforehand, it really is not necessary...

Reply:
it won't give us the node name, would it?
because the message is declared before we have a "node", so we won't really know in which node the upgrade started

Reply:
No, node name won't be added.

Reply:
so not doing it (for not having the node's name), but also because with using `log_run_info` we won't see it annotated in Grafana